optaplanner-webexamples/pom.xml

```diff
@@ -157,6 +157,12 @@
       <groupId>org.jboss.resteasy</groupId>
       <artifactId>resteasy-jaxrs</artifactId>
       <scope>test</scope>
+      <exclusions>
+        <exclusion>
```

Comment:
1) In the latest version, I don't see the commons-logging dep any more
  http://central.maven.org/maven2/org/jboss/resteasy/resteasy-jaxrs/3.0.17.Final/resteasy-jaxrs-3.0.17.Final.pom
As we're moving to EAP 7/ Wildfly 11, I presume we 'll be using that version?

2) Can we do these kind of exclusions on a higher level, such as ip-bom or kie-parent?